xds/data/orca/v3/orca_load_report.proto

```diff
@@ -21,7 +21,7 @@ message OrcaLoadReport {
 
   // Memory utilization expressed as a fraction of available memory
   // resources. This should be derived from the latest sample or measurement.
-  double mem_utilization = 2 [(validate.rules).double.gte = 0, (validate.rules).double.lte = 1];
+  double mem_utilization = 2 [(validate.rules).double.gte = 0];
```

Comment:
The value may be larger than 1.0 when the usage exceeds the reporter dependent notion of soft limits, like the `application_utilization` did.

Reply:
FWIW, to the best of my knowledge, the PGV validation rules are not actually used for validation at runtime, but just as a guideline to the implementers.

Reply:
right, the validation only works if the call the validate method 🤔

Reply:
FWIW, I this there should be a limit that is enforced in the code, regardless of whether the thresholds are explicitly stated here or not.
I would suggest that the code caps the range to something like [0, 1], just to prevent bad upstream actors from the ability to impact the load-balancing decision too much (either intentionally or unintentionally).